qi-doc/scribblings/list-operations.scrbl

```diff
@@ -0,0 +1,76 @@
+#lang scribble/doc
+@require[scribble/manual
+	 (for-label racket/list
+	 	    racket/base)]
```

Comment:
Indentation: raw tabs?
```suggestion
@require[scribble/manual
         (for-label racket/list
                    racket/base)]
```

Reply:
Will do after merging this into the integration branch.